docs/restapi-endpoints.md

```diff
@@ -209,7 +209,7 @@ This section provides a detailed list of avaliable endpoints in Broker REST API.
   Delete a cluster reference of the Asperathos section
 
 * **URL**: `/submissions/cluster/:cluster_name/delete`
```

Comment:
I think is better change the contract, removing the "/delete", because in this case is redundant.